platform/mellanox/mlnx-platform-api/sonic_platform/component.py

```diff
@@ -133,6 +135,17 @@ class ONIEUpdater(object):
 
     ONIE_IMAGE_INFO_COMMAND = '/bin/bash {} -q -i'
 
+    BIOS_UPDATE_FILE_EXT = '.rom'
+
+    def __add_prefix(self, image_path):
```

Comment:
@alexrallen what is the motivation for this change? What is the use case? IMHO, looks like a hack.

Reply:
I can't say I love it either I am more than happy to get your thoughts.

ONIE has to be installed before BIOs during boot and the installer processes the files in alphabetical order.

So I append 000 to the ONIE name so no matter what it is named it will always install first.